api/tests/unit/projects/code_references/test_unit_projects_code_references_views.py

```diff
@@ -1,24 +1,18 @@
 import freezegun
-from common.projects.permissions import VIEW_PROJECT
 from rest_framework.test import APIClient
 
 from features.models import Feature
 from projects.code_references.models import FeatureFlagCodeReferencesScan
 from projects.models import Project
-from tests.types import WithProjectPermissionsCallable
 
 
 @freezegun.freeze_time("2025-04-14T09:30:00-0300")
 def test_CodeReferenceCreateAPIView__responds_201_with_accepted_code_references(
+    admin_client_new: APIClient,
     project: Project,
-    staff_client: APIClient,
-    with_project_permissions: WithProjectPermissionsCallable,
 ) -> None:
-    # Given
-    with_project_permissions([VIEW_PROJECT])  # type: ignore[call-arg]
-
     # When
-    response = staff_client.post(
+    response = admin_client_new.post(
```

Comment:
This test used to test the minimal permissions needed for a 201 - now it's only testing that it works for an admin user. Perhaps later tests are covering some of this instead, but I don't love this modification.

Reply:
It looks like the rest of the tests do cover the negative cases, so maybe it's fine.

Reply:
> Perhaps later tests are covering some of this instead, but I don't love this modification.

I did an experiment ([described above](https://github.com/Flagsmith/flagsmith/pull/5979#discussion_r2299335658)) that may help, though I'm not certain. Since this only affects this PoC, I'll merge this as-is and hand it over to the frontend team in their timezone. Let's please revisit this.